static/app/app/lib/form-view/index.js

```diff
@@ -35,7 +35,7 @@ InputWriterSet.registerDefault(function(el, value){
     }
 });
 
-InputReaderSet.registerDefault(function(el){    
```

Comment:
There's a setting on Atom that automatically removes extra whitespace from files when they are opened. While this in general good, it's annoying for our workflow because it makes files look like they have been edited when in fact they have not been (such as this file). @cisacke and @josefdaly make sure to turn the setting off.